src/wordlift/mappings/pages/class-edit-mappings-page.php

```diff
@@ -242,12 +247,34 @@ private static function get_post_type_key_and_value() {
 				)
 			);
 		}
+
 		return array(
 			'post_type_option_name' =>  $post_type_option_name,
 			'post_type_option_values' => $post_type_option_values
 		);
 	}
 
+    /**
+     * Return post type option and post type option values.
+     *
+     * @return array Array of post_type_option and post_type_option_values.
+     */
+    private static function get_post_taxonomy_key_and_value() {
+
+        $post_taxonomy_option_name   = array(
+            'label'      => __( 'Post Taxonomy', 'wordlift' ),
+            'value'      => Wordlift\Mappings\Validators\Post_Taxonomy_Term_Rule_Validator::POST_TAXONOMY,
+            // Left empty since post types are provided locally.
```

Comment:
this comment can be removed, and i think api_source should be set to post_taxonomy